containers/podman-desktop-extension-ai-lab
# chore(tsconfig): enabling strictNullChecks

### What does this PR do?

Enabling the `strictNullChecks` and fixing the places where it raised an error

### Screenshot / video of UI



### What issues does this PR fix or reference?

Fixes https://github.com/containers/podman-desktop-extension-ai-lab/issues/811

### How to test this PR?

- [x] pipeline should be ✅ 

⚠️ Please do try some basic scenarios, this should not change much, but we are never too much careful
